### PR TITLE
Be consistent in icons colors

### PR DESCRIPTION
**Pull request by dmarcoux**
Follow-up to #5713 after our discussions in the review meeting this morning.

icons

### CODE REVIEW

File: src/api/app/views/webui2/webui/attribute/index.html.haml [hunk 1]
```diff
@@ -31,18 +31,18 @@
...
+                      %i.fas.fa-times-circle.text-danger
     - else
       %p
         %em No attributes set
 
     - if policy(@package).update?
       %p
         = link_to(new_attribs_path(project: @project.name, package: @package), title: 'Add a new attribute') do
-          %i.fas.fa-plus-square
+          %i.fas.fa-plus-square.text-primary
```
Comment (DavidKang) [inline src/api/app/views/webui2/webui/attribute/index.html.haml]: not sure if it should be `square` or `circle` :thinking:

Comment (dmarcoux) [reply src/api/app/views/webui2/webui/attribute/index.html.haml]: I am fine with using `circle`, to be consistent with `package#show`.

**Comment by dmarcoux** [comment]
Fails are on master, we need to fix this...

**Comment by dmarcoux** [comment]
[Bootstrap: use colors for icons on Package#attribute page](https://trello.com/c/jLHS9VXF/1556-bootstrap-use-colors-for-icons-on-packageattribute-page)

**Pull request merged**